### PR TITLE
fix: Pro套件启动报错, 菜单优化, 国际化词条未生效

### LINKED ISSUE #87: 🐛 [Bug]: TinyPro Vue 部分国际化词条未生效
### Version

latest

### node-version

latest

### Link to minimal reproduction

如下图所示：
image

### Step to reproduce

访问链接：https://opentiny.design/vue-pro/pages/board/home

image

### What is expected

_No response_

### What is actually happening

_No response_

### Any additional comments (optional)

_No response_

### LINKED ISSUE #91: 🐛 [Bug]:
### Version

最新的

### node-version

19.0.0

### Link to minimal reproduction

_No response_

### Step to reproduce

初始化完毕之后直接运行即可，点击右侧菜单

### What is expected

高亮选中的应该是路由变化的那个

### What is actually happening

右边菜单最外层的点击 路由没有变化，不应该出现选中效果，和ng版本的一致

### Any additional comments (optional)

_No response_

### LINKED ISSUE #92: Pro套件初始化后启动报错🐛 [Bug]:
### Version

v1.1.0

### node-version

v18.17.1

### Link to minimal reproduction

_No response_

### Step to reproduce

按照Pro套件文档操作，不选择数据库，不选择使用后台创建一个工程，最后调用tiny start 启动后报错。

### What is expected

_No response_

### What is actually happening

_No response_

### Any additional comments (optional)

_No response_

### PR DESCRIPTION
**Pull request by jishuai-cool**
# PR
fix #87
fix #91
fix #92 
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-cli/blob/main/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/toolkits/pro/template/tinyvue/config/vite.config.base.ts [hunk 1]
```diff
@@ -41,7 +41,7 @@ const config = {
       },
       {
         find: 'vue-i18n',
-        replacement: 'vue-i18n/dist/vue-i18n.cjs.js', // Resolve the i18n warning issue
+        replacement: 'vue-i18n/dist/vue-i18n.esm-bundler.js', // Resolve the i18n warning issue
```
Comment (fengyon) [inline packages/toolkits/pro/template/tinyvue/config/vite.config.base.ts:44]: 这个改了之后，把 package.json中 vue-i18n的依赖版本升级一下, `^9.3.0`

File: packages/toolkits/pro/template/tinyvue/src/components/navbar/index.vue [hunk 2]
```diff
@@ -25,7 +25,8 @@
         <div class="divider"></div>
       </li>
       <li @click="changeLan">
-        <span>{{ i18.locale.value }}</span>
+        <span v-if="i18.locale.value == 'zhCN'">中文</span>
```
Comment (hello-yezi) [inline packages/toolkits/pro/template/tinyvue/src/components/navbar/index.vue]: 使用 ===

**Comment by fengyon** [comment]
关联一下修复的问题单吧

在这个PR描述顶部的”PR“下面加一下

fix #issueNumber

**Review by fengyon — APPROVED** [review]
检视通过

**Pull request merged**